fabric-tool-attributes-v1/build.gradle

```diff
@@ -0,0 +1,8 @@
+archivesBaseName = "fabric-tool-attributes-v1"
```

Comment:
Imo we should stick with singular names.

Reply:
`fabric-tool-attribute-v1` makes no sense, and it's Java code style to use plurals where necessary. It's necessary here, because each tool has multiple attributes.